internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1175,8 +1221,17 @@ func (c *ClusterManager) reconcileClusterMachinePools(cluster api.Cluster) (bool
 		if err != nil {
 			return false, err
 		}
+		dynamicCapacityInfo[supportedInstanceType] = api.DynamicCapacityInfo{
+			MaxNodes: int32(machinePoolRequest.AutoScaling.MaxNodes),
+		}
 	}
 
+	// update the dyamic capacity info for them to be stored in the database
+	_ = cluster.SetDynamicCapacityInfo(dynamicCapacityInfo)
+	srvErr := c.ClusterService.Update(cluster)
+	if srvErr != nil {
+		return true, errors.Wrapf(srvErr, "failed to update cluster")
```

Comment:
Shouldn't return `false` on error?

Reply:
Good point. It's not noticeable since the errors are checked first but I'll change it to false to avoid any confusion.

Reply:
updated